pkg/aspect/accessLogsManager.go

```diff
@@ -117,14 +118,29 @@ func (e *accessLogsWrapper) Close() error {
 }
 
 func (e *accessLogsWrapper) Execute(attrs attribute.Bag, mapper expr.Evaluator, ma APIMethodArgs) Output {
-	labels, err := evalAll(e.labels, attrs, mapper)
-	if err != nil {
-		return Output{Status: status.WithError(fmt.Errorf("failed to eval labels for log %s with err: %s", e.name, err))}
+
+	labels := make(map[string]interface{}, len(e.labels))
+	for name, exp := range e.labels {
+		v, err := mapper.Eval(exp, attrs)
+		if err == nil {
+			labels[name] = v
+			continue
+		}
+		if name == "timestamp" {
```

Comment:
I actually added in typechecking this morning, so what I described is possible now, just not during execute time. AFAIK we don't ever plan on introducing descriptors into the exec path, so it likely will never be possible to do what I described. We'll have to noodle on this a bit to come up with a general solution, but no reason for that to block this PR.